aws/rapids_sagemaker_hpo.ipynb

```diff
@@ -642,7 +642,7 @@
    "metadata": {},
```

Comment:
<p>I'm guessing we still need to pull and push to ECR, since we can't directly pull from DockerHub to SageMaker</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>